test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -85,8 +85,14 @@ public void testLoadResponseGeneratorModeOfOpNoSuchFactoryMethod() {
     for (Map.Entry<String, String> entry : configEntries.entrySet()) {
       TestHelper.setConfigValue(config, entry.getKey(), entry.getValue());
     }
-    thrown.expect(InvalidConfigurationException.class);
-    DatabaseAdaptor.loadResponseGenerator(config);
+    try {
```

Comment:
There's no need for the JUnit 3 idiom here. You can just add 

thrown.expectMessage("...");

which uses String.contains to look for a match.